editor/components/EMSLinkRecordButton.tsx

```diff
@@ -14,37 +16,58 @@ export interface EMSLinkRecordButtonProps extends Record<string, unknown> {
 
 const EMSLinkRecordButton: React.FC<
   RowActionComponentProps<EMSPatientCareRecord, EMSLinkRecordButtonProps>
-> = ({ record, additionalProps }) => {
-
+> = ({ record, additionalProps, mutation, onSaveCallback }) => {
   const isLinkingInProgress = !!additionalProps?.selectedEmsPcr;
   const isLinkingThisRecord =
     additionalProps?.selectedEmsPcr &&
     record.id === additionalProps?.selectedEmsPcr.id;
 
-  const isRecordAlreadyLinked = !!record.person_id;
-
-  if (isRecordAlreadyLinked) {
-    return null;
-  }
+  const { mutate: setRecordManualUnmatched } = useMutation(mutation);
 
   return (
     <PermissionsRequired allowedRoles={allowedLinkRecordRoles}>
-      <Button
-        size="sm"
-        variant={isLinkingThisRecord ? "secondary" : "primary"}
-        disabled={isLinkingInProgress && !isLinkingThisRecord}
-        onClick={() => {
-          additionalProps?.onClick(record);
-        }}
-      >
-        {!isLinkingThisRecord && (
-          <AlignedLabel>
-            <FaLink className="me-2" />
-            <span>Select person</span>
-          </AlignedLabel>
-        )}
-        {isLinkingThisRecord && <span>Cancel</span>}
-      </Button>
+      <div className="d-flex">
+        <Button
+          size="sm"
+          variant={isLinkingThisRecord ? "secondary" : "primary"}
+          disabled={isLinkingInProgress && !isLinkingThisRecord}
+          onClick={() => {
+            additionalProps?.onClick(record);
+          }}
+        >
+          {!isLinkingThisRecord && (
+            <AlignedLabel>
+              <FaLink className="me-2" />
+              <span>Select person</span>
+            </AlignedLabel>
+          )}
+          {isLinkingThisRecord && <span>Cancel</span>}
+        </Button>
+        <Dropdown className="ms-1">
+          <Dropdown.Toggle variant="outline-primary" size="sm">
+            <FaEllipsisVertical />
+          </Dropdown.Toggle>
+          <Dropdown.Menu renderOnMount popperConfig={{ strategy: "fixed" }}>
```

Comment:
this popperConfig setting made sure the dropdown wasnt rendering inside of the card causing it to be cutoff sometimes and you had to scroll down in the card to see it. the `renderOnMount` fixed a bug where the dropdown was rendering on the corner of the screen when you first click it